### PR TITLE
Aqa/js conditions loops

### PR DESCRIPTION
**Pull request by manatskaya**
Hi Luibov,
This is pull request to lesson 5. Please review once you have time.
Thank you
Svitlana

### CODE REVIEW

File: js-conditions-loops/conditionsAndloops.js [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+    console.log('->', currentLetter);
+    
+    if (currentLetter === 'a') {
+        continue;
+    }
+
+    if (currentLetter === 'n') {
+        break;
+    }
+}
```
Comment (landriiuk) [inline js-conditions-loops/conditionsAndloops.js]: This loop is not working properly. 
Screenshot 2023-09-29 at 17 36 08

File: js-conditions-loops/conditionsAndloops.js [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+        console.log('October');
+        break;
+    case 11:
+        console.log('November');
+        break;
+    case 12:
+        console.log('December');
+        break;
+    default:
+            console.log('new year started')
```
Comment (landriiuk) [inline js-conditions-loops/conditionsAndloops.js]: missed ";"

**Comment by landriiuk** [comment]
Pls, fix task 5 using newstring[index]. 
your iterator(index++) should be at the bottom.

**Comment by manatskaya** [comment]
Hi Luibov,
Changes have been added accordingly to the comment.
Thank you
Svitlana

**Pull request merged**